src/Stoa.ts

```diff
@@ -10,6 +11,17 @@ class Stoa {
         this.stoa.use(bodyParser.urlencoded({ extended: false }))
         // parse application/json
         this.stoa.use(bodyParser.json())
+        // DB fileName
+        var filename: string = ".cache";
+        // create blockStorage
+        var blockStorage: BlockStorage = new BlockStorage(filename, (err: any) =>
+        {
+            if (err != null)
+            {
+                console.error("Failed to open database");
+                return;
```

Comment:
It's a simple return... So the code continues as if nothing happened ?

Reply:
Stoa express will not be able to provide any service.
The router does not open.

Reply:
The application should stop. Can this be made to throw ?

Reply:
I think we should make a exception .